examples/timeseries/timeseries_classification_from_scratch.py

```diff
@@ -224,4 +224,10 @@ def make_model(input_shape):
 after 200 epochs. Beyond the 200th epoch, if we continue on training, the validation
 accuracy will start decreasing while the training accuracy will continue on increasing:
 the model starts overfitting.
+
+Example available on HuggingFace:
```

Comment:
I think that you need a break between the intro sentence (`Example available on HuggingFace:`) and the table, to make the formatting work. Can you add that and regenerate?